2-mandatory/3-stocks.js

```diff
@@ -34,7 +34,18 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
```

Comment:
Great job!
Now that you have something that works - maybe you could split some of the code into a separate function improve readability. For example, you could have a function which just calculates the average price given the `stockPricesLastFiveDays` array.